src/s390x/zipl.rs

```diff
@@ -86,9 +79,12 @@ fn run_zipl<P: AsRef<Path>>(
                 copy(src.path(), blsdir.join(src.file_name())).unwrap();
             });
         let mut extra = vec!["ignition.firstboot".to_string()];
-        if let Some(kargs) = firstboot_kargs {
+        let firstboot_kargs = std::fs::read_to_string(firstboot_file)
+            .context("reading firstboot kargs from ignition.firstboot")?;
+        let firstboot_kargs = firstboot_kargs.trim();
```

Comment:
The `ignition.firstboot` file [isn't that simple](https://github.com/coreos/coreos-installer/blob/d14fe5dbb5a864eee4d45070a6a7eb3943b54249/src/install.rs#L307), unfortunately.

Reply:
Ahhh of course. There's no easy way to solve this generically, but at least we can special-case the format we expect. Fix added!